c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,10 +80,13 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
-    }));
+    var mapSize = this.map.getSize();
+    if (mapSize) {
+      this.map.getView().fit(app.MapController.DEFAULT_EXTENT, mapSize,
+      /** @type {olx.view.FitOptions} */ ({
```

Comment:
Is this cast required? There is no cast in https://github.com/c2corg/v6_ui/pull/340/files#diff-2fb178a58cd1af945acd73ff3c2b5ca7R172